File: CS/E965.Module/BusinessObjects/FileSystemStoreObjectDemo.cs
using DevExpress.Xpo;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using FileSystemData.BusinessObjects;

namespace E965.Module.BusinessObjects {
    [DefaultClassOptions]
    [FileAttachment("File")]
    public class FileSystemStoreObjectDemo : BaseObject {
        public FileSystemStoreObjectDemo(Session session) : base(session) { }
        [Aggregated, ExpandObjectMembers(ExpandObjectMembers.Never), ImmediatePostData]
        public FileSystemStoreObject File {
            get { return GetPropertyValue<FileSystemStoreObject>("File"); }
            set { SetPropertyValue<FileSystemStoreObject>("File", value); }
        }
    }
}

File: VB/E965.Module/BusinessObjects/FileSystemStoreObjectDemo.vb
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Xpo
Imports FileSystemData.BusinessObjects

Namespace E965.Module.BusinessObjects
	<DefaultClassOptions, FileAttachment("File")>
	Public Class FileSystemStoreObjectDemo
		Inherits BaseObject

		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
		<Aggregated, ExpandObjectMembers(ExpandObjectMembers.Never), ImmediatePostData>
		Public Property File() As FileSystemStoreObject
			Get
				Return GetPropertyValue(Of FileSystemStoreObject)("File")
			End Get
			Set(ByVal value As FileSystemStoreObject)
				SetPropertyValue(Of FileSystemStoreObject)("File", value)
			End Set
		End Property
	End Class
End Namespace
